tests/fw_rules_tests/policies/expected_output/semantic_diff_with_different_topologies-scheme_output.csv

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Why do we get this redundant new line?

Reply:
1. In NetworkConfigQueryRunner.py a '\n' is always added after calling execution of a single query 
`output += query_output + '\n'`
2. SemanticDiff is one of the 2 queries that support output formats other than .txt 
3. In this specific test (fw_rules_tests\policies\semantic_diff_with_different_topologies-scheme.yaml) the first query (semantic_diff_identical) has a "no policy" as its first config, 
so for the **.txt** format it returns a query_output = no_policy and policy are semantically equivalent. (printed at first line in semantic_diff_with_different_topologies-scheme_output.txt ), but returns an empty query_output for other formats (.csv, .yaml , .md ..) - so we see an empty line in these files 

If needed, I may add an if condition in the NetworkConfigQueryRunner.py before adding the  "+ '\n'"
or adjust the exec method of semanticDiff  query to return something that could be printed in the other formats instead of empty output.